WebUI/plugins/qrc-generator.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/*
+  The QrcGeneratorPlugin for Webpack generates a Qt resource file
+  listing all the assets produced by Webpack.
+
+  This allows embedding of a webpack-generated bundle into a Qt-based
+  C++-application.
+
+  Qt's WebEngine module supports loading websites from QRC resources
+  out-of-the-box. So this is a convenient way to embed a static webpage.
+*/
+class QrcGeneratorPlugin {
+    apply(compiler) {
+        const onEmit = (comp) => {
+            const prefix = '<!DOCTYPE RCC>\n<RCC version="1.0">\n<qresource prefix="/WebUI">\n';
```

Comment:
The QRC files generated by QT Designer don't have the doctype and version tag - which is probably not an issue, just wanted to point that out.

For readability, maybe add a '\t' before the <qresource> tag and '\t\t' before each file, or 4 spaces?
Edit: I see you're doing that for each file, so I'd add another indentation level.

Reply:
Sure. I added two more leading whitespaces for the qresource-level.

Regarding the format: I was following the official docs (https://doc.qt.io/qt-5/resources.html) which suggest a format which includes the version and the doctype. So I would rather stick to that.